src/DotLiquid/Context.cs

```diff
@@ -3,6 +3,9 @@
 using System.Collections.Generic;
 using System.Globalization;
 using System.Linq;
+#if NETCore
+using System.Reflection;
+#endif
```

Comment:
Required because in corefx, Advanced type operations are implemented with extensions.